development/build/index.js

```diff
@@ -0,0 +1,125 @@
+const gulp = require('gulp')
+const gulpZip = require('gulp-zip')
+const livereload = require('gulp-livereload')
+const del = require('del')
+const { promises: fs } = require('fs').promises
```

Comment:
I'm guessing this is what was intended :sweat_smile: 
```suggestion
const { promises: fs } = require('fs')
```